src/main/java/com/github/martynagil/drugstoremanagement/model/Employee.java

```diff
@@ -113,7 +117,8 @@ public Shop getShop() {
         return workTimes;
     }
 
-    public void dismiss(LocalDate dateOfDismissal) {
-        this.dateOfDismissal = dateOfDismissal;
+    private boolean isNotDissmised() {
```

Comment:
raczej nie robi się `not` w metodzie, `isDissmised()` jest trochę prostsza w rozumowaniu, dlatego odwróciłbym ten warunek